Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -34,6 +34,21 @@ open class InputTextView: UITextView, AppearanceProvider {
         }
     }
     
+    /// The minimum height of the text view.
+    /// When there is no content in the text view OR the height of the content is less than this value,
+    /// the text view will be of this height
+    open var minimumHeight: CGFloat = 38.0
```

Comment:
I posted into our channel message that we should probably think more of using stored properties vs read-only properties and overriding them, IMHO this is exactly the case where we want to use get-only property so it's easier to override when someone subclasses `InputTextView`.

Reply:
```suggestion
   open var minimumHeight: CGFloat {
       return 38.0
   }
```

Reply:
💯  true. I'll change this